init.lua

```diff
@@ -60,6 +60,29 @@ local source = string.sub(debug.getinfo(1,'S').source, 2)
 local path = string.sub(source, 1, string.find(source, "/[^/]*$"))
 local noicon = path .. "noicon.png"
 
+local function cycle(altTabTable, altTabIndex, dir)
```

Comment:
I see the problem with the invocation of `cycle(...)` in the `switch(...)` block. Well done!